domain/learningandworkprogress/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/learningandworkprogress/induction/dto/CreatePreviousWorkExperiencesDto.kt

```diff
@@ -5,6 +5,7 @@ import uk.gov.justice.digital.hmpps.domain.learningandworkprogress.induction.Wor
 
 data class CreatePreviousWorkExperiencesDto(
   val hasWorkedBefore: HasWorkedBefore,
+  val hasWorkedBeforeNotRelevantReason: String?,
```

Comment:
My first PR never actually mapped the new field in the Create or Update DTOs; so the new field, whilst on the API request, was never actually mapped and written to the database! 🤦‍♂️